org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/interpreter/AbstractConstantExpressionsInterpreter.xtend

```diff
@@ -202,15 +202,16 @@ class UnresolvableFeatureException extends ConstantExpressionEvaluationException
 	
 }
 
+@Data
```

Comment:
is there a reason for adding this?

Reply:
The classes' fields were unused, adding an @Data adds accessors to the fields making them non-useless

Reply:
👍

Reply:
Hmm JvmField would not be serializable but exceptions are serializable. I'd rather remove the field and set the root cause to be the given cause.

Reply:
I'll fix this sometime next week